### ISSUE TITLE
Capability to recover unsaved file data after crash or OS restart

### ISSUE THREAD

**Issue by saithalavi** [geany/geany#1603]
The unsaved files will not be available after an OS restart or crash. This behavior is seen in Mac, not sure about Windows. It's one capability I feel missing in Geany compared to Notepad++ in Windows and TextWrangler in Mac.

**Comment by elextr** [comment]
Try looking at the `save actions` plugin for some intermediate saving features.

**Comment by elextr** [comment]
@hellyberry yes its not saved until (proper) quit.  If someone wants tomake a well written PR to add it to save actions (with option to disable) it probably would be accepted.
